sdk/resourcemanager/Azure.ResourceManager/api/Azure.ResourceManager.netstandard2.0.cs

```diff
@@ -218,7 +218,7 @@ public partial class ManagementGroupCreateOrUpdateContent
         public string DisplayName { get { throw null; } set { } }
         public string Id { get { throw null; } }
         public string Name { get { throw null; } set { } }
-        public string ResourceType { get { throw null; } }
```

Comment:
Per the [description](https://github.com/Yao725/azure-rest-api-specs/blob/main/specification/managementgroups/resource-manager/Microsoft.Management/stable/2021-04-01/management.json#L1796-L1799) in swagger, we can know this property is a truly `ResourceType`